index.bs

```diff
@@ -14,15 +14,15 @@ Former Editor: Rich Tibbett, Opera Software ASA
 Former Editor: Tim Volodine, Google Inc
 Former Editor: Steve Block, Google Inc until July 2012
 Former Editor: Andrei Popescu, Google Inc until July 2012
-Abstract: This specification defines several new DOM events that provide information about the physical orientation and motion of a hosting device.
-Boilerplate: omit issues-index, omit conformance, repository-issue-tracking no
+Abstract: This specification defines events that represent the physical orientation and motion of a hosting device. These events provide web applications with access to orientation and motion data. The specification is designed to be agnostic to the underlying sources of this data, aiming to achieve interoperability across different environments.
+Boilerplate: omit issues-index, repository-issue-tracking no
 Include MDN Panels: if possible
 Implementation Report: https://wpt.fyi/results/orientation-event
-Issue Tracking: DeviceOrientation Event Specification Issues Repository https://github.com/w3c/deviceorientation/issues
-Status Text: <p>This specification was initially published as a <a href="https://www.w3.org/TR/2016/CR-orientation-event-20160818/">Candidate Recommendation</a> in August 2016 and was <a href="https://www.w3.org/TR/2017/NOTE-orientation-event-20170530/">retired</a> in 2017 due to the Geolocation Working Group closure. In 2019 Devices and Sensors Working Group adopted this specification and during 2019-2024 made substantial interoperability, test automation, privacy and editorial improvements as outlined in the <a href="#changes">Changes</a> section. These changes aligned the specification with widely available implementations. In 2024 this specification became a joint deliverable between the Devices and Sensors Working Group and WebApps Working Group.</p>
```

Comment:
As discussed earlier, and per feedback received during wide review, this Status Text provides important information to readers and must not be removed. It informs readers of significant changes and provides pointers for more information. Web specifications are written to an audience broader than implementers and not all want to go down to the commit log level to understand the important changes in the specification's evolution.

Reply:
I think we have to agree to disagree. 

This is an editorial decision and it doesn't seem to add things that can't be gleamed from looking at the change log. It just adds bloat.  

I'm happy to change it but you need to propose something better: why would anyone care when the Geolocation WG closed etc. etc. all the text is not helpful and when it became a joint deliverable. It adds zero value.

Reply:
As an editor you are expected to reflect group decisions correctly in the documents you are editing, and be extra careful when working on a document that is about to be published as a CR. As an editor you are authoring a document not for yourself but to a wider audience. If you think some words are inappropriate on this line we can try to find alternatives, but you need to provide more information so we can understand the issue.

To elaborate, the intent of this text is to help the wider audience understand the following:

This document has had multiple deliverers and the earlier design discussions can be found not from this GH repo (as many readers would expect) but from the Geolocation WG mailing list archives, interesting discussions e.g. in:

https://lists.w3.org/Archives/Public/public-geolocation/2010Mar/
https://lists.w3.org/Archives/Public/public-geolocation/2010Apr/
https://lists.w3.org/Archives/Public/public-geolocation/2010Jul/

We could possibly link to https://lists.w3.org/Archives/Public/public-geolocation/ directly to further help with discovery of this information that explains why some of the uncommon design decisions were made back then.

In addition, this text provides an overview of key changes since the CR 2016 and points to a changes summary for people who don't want to look and parse the noisy commit log from the last 8 years. (We may be able to automative this with a summarization model in the future.)

In general I encourage all group participants to be aware of the history and earlier discussions, so we can collectively learn from the past. Making this information more accessible helps with that. It is quite a nice feature actually that also this comment that I make will be forever archived for future generations to see in the same archive as those discussion from 2010 when this API was started.

Reply:
I personally suppose texts for adoption or any other contribution could move to independent 'Acknowledgements' section...

Reply:
>As an editor you are authoring a document not for yourself but to a wider audience. 

I've been doing this for almost 20 years, but thank you for telling me how to do my job. 

> further help with discovery of this information that explains why some of the uncommon design decisions were made back then.

Right, but this is completely hypothetical. Has this ever come up in practice? Also, this seems like an extremely niche audience requirement. 

>In addition, this text provides an overview of key changes since the CR 2016 and points to a changes summary for people who don't want to look and parse the noisy commit log from the last 8 years.

That's in the "Changes" section. Not here.  This just "during 2019-2024 made substantial interoperability, test automation, privacy and editorial improvements", which is literally what every single spec does throughout its lifecycle. 

The rest could probably go in a wiki or something. It still doesn't strike me as something that should be eating up space at the top of the document. 

@himorin wrote
 > I personally suppose texts for adoption or any other contribution could move to independent 'Acknowledgements' section...

Yeah, that seems more appropriate.

Reply:
This is to satisfy [Transition Requirements for Candidate Recommendation Snapshot](https://www.w3.org/Guide/transitions/?profile=CR&cr=new):

>Include a link to a change log where new features are highlighted, highlight them in the Status of the Document.

Reply:
Right, but the current barely highlights any new features. Only the test automation might be relevant. I'll gather something better.

Reply:
we may add lines of link to #changes in bikeshed boilerplate? e.g.
https://github.com/speced/bikeshed-boilerplate/blob/69a4a2fdec4e7032f542ae9cc99c8773fce3d4ab/boilerplate/immersivewebwg/status-CR.include#L44

Reply:
yes, that works. Let's add that... I'll just working on the new status text.

Reply:
@himorin, actually, that assumes all specs would have a Changes section, which might not be the case. Let's do it on per spec basis.